cmd/lock-rpc-server.go

```diff
@@ -269,11 +269,10 @@ func (l *lockServer) lockMaintenance(interval time.Duration) {
 			// - some network error (and server is up normally)
 			//
 			// We will ignore the error, and we will retry later to get resolve on this lock
-			log.Println("  Dsync.Active failed:", err)
+			log.Infoln("Dsync.Active failed:", err)
```

Comment:
This is not an error perse, in fact it can legitimately happen while either a server to be contacted is down or the network is down (and during a subsequent run of maintenance, this lock will be retried).

So this is more of an 'informative' message (and hence the 'Infoln'), or we should silently remove it altogether.